docker/Dockerfile-dhvirtualenv

```diff
@@ -70,7 +74,10 @@ RUN apt-get update -qq -o Acquire::Languages=none \
         python3-venv \
         sqlite3 \
         libpq-dev \
-        xmlsec1
+        xmlsec1 \
+    && ( env DEBIAN_FRONTEND=noninteractive apt-get install \
+         -yqq --no-install-recommends -o Dpkg::Options::=--force-unsafe-io \
+         dh-systemd || true )
```

Comment:
Per GitHub comment above, this basically says "try to install dh-systemd, but it's OK if it fails or doesn't exist."

dh-systemd exists as a dummy transitional package on all releases except Ubuntu xenial (sans backports), where we actually do need it. So we need to try to install it. However, the transitional package was removed in Debian Bullseye, so we have to gracefully continue even if the package does not exist.

...all this goes away when we drop Xenial support in ~2 months.